nodejs/week4/warmup.js

```diff
@@ -0,0 +1,53 @@
+class Email {
+    constructor(subject, body) {
+        this.subject = subject;
+        this.body = body
+    }
+}
+
+function getPercentUppercase(str, upperCaseChar) {
```

Comment:
This is a GREAT function name. Describes exactly what it does, well done!

Reply:
Thank you Benjamin :)